app/src/main/java/ua/kozlov/madfood/Groups.java

```diff
@@ -0,0 +1,50 @@
+package ua.kozlov.madfood;
+
+import android.content.Context;
+import java.util.ArrayList;
+
+public class Groups implements GroupsInterface {
+    private ArrayList<String> mGroupsList;
+    private String[] mGroupsNames = {"Vegetables", "Baked products",
+            "Beverages nonalcoholic", "Bread flour products", "Cannedfish, seafoods",
+            "Cannedfruits, vegetables, mushrooms", "Cannedeat products",
+            "Cerealgrains, cereals, flakes", "Cerealgrains cooked", "Fastfoods",
+            "Fish, seafoods", "Fruits", "Fruitsdried", "Icecream",
+            "Jams", "Legumes", "Meels entrees sidedishes", "Meats, byproducts",
+            "Mushrooms", "Nut and seed products", "Oils, fats",
+            "Poultry products", "Sausages", "Snacks", "Soups gravies",
+            "Soy products", "Sushi", "Sweets desserts"};
```

Comment:
@OlehKozlov is that init of array is correct, if user will want to add his own group??